docs/environment-variables.md

```diff
@@ -23,6 +23,7 @@ This document outlines environment variables that can be used to customize behav
 | `ARGO_AGENT_CPU_LIMIT`                   | `resource.Quantity` | `100m`                                                                                      | CPU resource limit for the agent.                                                                                                                                                                                                                                        |
 | `ARGO_AGENT_MEMORY_LIMIT`                | `resource.Quantity` | `256m`                                                                                      | Memory resource limit for the agent.                                                                                                                                                                                                                                     |
 | `ARGO_POD_STATUS_CAPTURE_FINALIZER`      | `bool`              | `false`                                                                                     | The finalizer blocks the deletion of pods until the controller captures their status.
+| `ARGO_TEMPLATE_WITH_INPUTS`      | `bool`              | `true`                                                                                     | Whether to keep large inputs section inside the environment variable of pods.
```

Comment:
I think `ARGO_TEMPLATE_WITH_INPUTS_PARAMETERS` is more semantically appropriate and it is better to write it clearly in the introduction to keep `inputs.parameters` in the `ARGO_TEMPLATE` environment variable.